roles/health_checks/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Include tasks
+  ansible.builtin.include_tasks: validation.yaml
+
+- name: Include tasks
+  ansible.builtin.include_tasks: "{{ operation.name }}.yaml"
```

Comment:
it should not have anything valid in terms of operations now , you can include health_checks.yaml task here

Reply:
but this `operation.name` is referring to the platform specific operations here like ios.yaml, iosxr.yaml & so on..